### ISSUE TITLE
How to handle special characters in item names

### ISSUE THREAD

**Issue by Kebap** [demonnic/muddler#9]
I have items names containing `:` or `/` characters for example, how to muddle?

**Comment by Kebap** [comment]
Testing through, putting : in file name does not seem to influence muddle after all.

Whereas / is a forbidden sign by OS and probably can't be used in item names then.

**Comment by demonnic** [comment]
Yeah, I should make a note of potential issues with special characters depending on OS/etc. I'm not doing any special handling there, apart from changing spaces to _ when converting name to .lua filename.

**Comment by Kebap** [comment]
> changing spaces to _ when converting name to .lua filename.

Interestingly spaces work fine for folder names which translate to folders in Mudlet with space characters alright
